src/pages/ProfilesExplorerView/components/SceneLabelValuesTimeseries/SceneTimeseriesMenu.tsx

```diff
@@ -140,7 +141,53 @@ export class SceneTimeseriesMenu extends SceneObjectBase<SceneTimeseriesMenuStat
       ) as TimeSeriesQuery;
   }
 
+  getFieldConfig() {
```

Comment:
Isn't it the same as https://github.com/grafana/traces-drilldown/pull/369? Could we just pass the data / fields and the investigation app would calculate the config?

Reply:
It is the same, yes. I wanted to avoid passing super large objects, which `data` could obviously be. Those objects are all formed into a Proxy and that makes working with those objects a bit more difficult. 

With a broader vision, we should really move the whole component into the investigations app and then use it in Drilldown apps with exposed components. That would simplify things for the Drilldown integrations.

Reply:
Do you think we could merge this never the less like it is?

Reply:
> It is the same, yes. I wanted to avoid passing super large objects, which data could obviously be.

What's the problem with passing big objects? 🤔  wouldn't it be actually easier if anyway we want to have a component inside investigations app?

> Do you think we could merge this never the less like it is?

If I need to maintain this code I need to double check again why we need these logic, it's not super obvious from reading the code. And we will probably need some tests to see a few examples.